service/prometheus/scrapeconfig.go

```diff
@@ -0,0 +1,61 @@
+package prometheus
+
+import (
+	"fmt"
+
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/config"
+	"k8s.io/client-go/pkg/api/v1"
+
+	"github.com/giantswarm/prometheus-config-controller/service/key"
+)
+
+// GetTarget takes a Kubernetes Service, and returns a LabelSet,
+// suitable for use as a target.
+func GetTarget(service v1.Service) model.LabelSet {
+	target := model.LabelSet{
+		model.LabelName(
+			fmt.Sprintf("%s.%s", service.Name, service.Namespace),
+		): "",
```

Comment:
The formatting is not that readable. Would be cool to separate the key creation and its actual usage.

Reply:
Done